app/controllers/application_controller.rb

```diff
@@ -1513,7 +1513,7 @@ def replace_gtl_main_div(options = {})
     render :update do |page|
       page << javascript_prologue
       page.replace(:flash_msg_div, :partial => "layouts/flash_msg")           # Replace the flash message
-      page << "miqSetButtons(0, 'center_tb');" # Reset the center toolbar
+      page << "sendDataWithRx({eventType: 'updateToolbarCount', countSelected: 0})"
```

Comment:
Actually, with https://github.com/ManageIQ/manageiq-ui-classic/pull/1058, you could just keep this `miqSetButtons` here. (But the rest of the changes are needed.)

Seems like refactoring everything around `miqSetButtons` is too bold for now, but the set-to-zero case should be safe.